cmd/pd-ctl/main.go

```diff
@@ -87,7 +87,9 @@ func loop() {
 			}
 			continue
 		}
-
+		if line == "exit" {
```

Comment:
if `InitPDClient` meet error, it will return error directly without execute the command. so I move `exit` to here.